kitsune/search/admin.py

```diff
@@ -48,6 +38,33 @@ class DeleteError(Exception):
     pass
 
 
+def create_batch_id():
+    """Returns a batch_id"""
+    # TODO: This is silly, but it's a good enough way to distinguish
+    # between batches by looking at a Record. This is just over the
+    # number of seconds in a day.
+    return str(int(time.time()))[-6:]
```

Comment:
I moved this over from another module. This is the only place in kitsune it gets used so might as well have it here.

The actual value doesn't matter a whole lot. If the way we generate it is an issue, we can change it--I don't really care one way or the other.